src/app/auth/auth.js

```diff
@@ -152,13 +90,10 @@ $(document).ready((function ($) {
 
     $('.appointment-button').click(function () {
```

Comment:
`$('.appointment-button').click(function () {` event has to be moved to `home.js` since you only have appointment form in home page.

Reply:
Done

Reply:
Resolved.